MoltenVK/MoltenVK/Vulkan/vulkan.mm

```diff
@@ -164,6 +164,8 @@ static inline void MVKTraceVulkanCallEndImpl(const char* funcName, uint64_t star
 // Define an extension call as an alias of a core call
 #define MVK_PUBLIC_VULKAN_CORE_ALIAS(vkf, ext)	MVK_PUBLIC_VULKAN_ALIAS(vkf##ext, vkf)
 
+#define MVK_PUBLIC_VULKAN_STUB(name, ret, ...) MVK_PUBLIC_VULKAN_SYMBOL ret name(__VA_ARGS__) { assert(false); }
```

Comment:
I know it's a little late, but I didn't notice this PR until now.

I wonder what would happen if instead, we used `__builtin_unreachable()` in these stubs.

Reply:
> I wonder what would happen if instead, we used __builtin_unreachable() in these stubs.

Hmmm...it would certainly be an interesting compiler optimizer experiment. But what could the compiler effectively do to optimize here? We certainly don't want it removing the entire function as uncallable, since that would defeat the purpose of adding the stubs in the first place.

If anyone wants to burn cycles on it for curiosity's sake...feel free. 😉